zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/PostPublisher.java

```diff
@@ -327,22 +318,25 @@ private static void reindexPublishingSearch(Collection collection) throws IOExce
             for (PendingDelete pendingDelete : collection.description.getPendingDeletes()) {
 
                 ContentTreeNavigator.getInstance().search(pendingDelete.getRoot(), node -> {
-                    logDebug("Deleting index from publishing search ").addParameter("uri", node.uri).log();
+                    info().data("uri", node.uri).log("Deleting index from publishing search ");
                     pool.submit(() -> {
                         try {
                             Indexer.getInstance().deleteContentIndex(node.type, node.uri);
                         } catch (Exception e) {
-                            logError(e, "Exception reloading search index:").log();
+                            error().logException(e, "Exception reloading search index:");
                         }
                     });
                 });
             }
 
-            logInfo("Redindex search completed").collectionId(collection)
-                    .timeTaken((System.currentTimeMillis() - start)).log();
+            info().data("collectionId", collection.getDescription().getId())
+                    .data("timeTaken", (System.currentTimeMillis() - start))
+                    .log("Redindex search completed");
 
         } catch (Exception exception) {
-            logError(exception, "An error occurred during the search reindex").collectionId(collection).log();
+            error().data("collectionId", collection.getDescription().getId())
+                    .logException(exception, "An error occurred during the search reindex");
+
```

Comment:
👇 remove `ExceptionUtils.printRootCauseStackTrace(exception)` its not needed anymore as we are logging the exception stack trace.